staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/types_jsonschema.go

```diff
@@ -161,6 +161,67 @@ type JSONSchemaProps struct {
 	//      Atomic maps will be entirely replaced when updated.
 	// +optional
 	XMapType *string `json:"x-kubernetes-map-type,omitempty" protobuf:"bytes,43,opt,name=xKubernetesMapType"`
+
+	// x-kubernetes-validations describes a list of validation rules written in an expression language.
```

Comment:
what is *an* expression language? Isn't it *the* CRD Expression Language?

Reply:
Yes, my bad. Will fix.